executor/ddl.go

```diff
@@ -179,6 +191,26 @@ func (e *DDLExec) executeDropDatabase(s *ast.DropDatabaseStmt) error {
 	return errors.Trace(err)
 }
 
+var systemTables = map[string]struct{}{
+	"user":                 {},
+	"db":                   {},
+	"tables_priv":          {},
+	"columns_priv":         {},
+	"gc_delete_range":      {},
+	"gc_delete_range_done": {},
+}
+
+func isSystemTable(schema, table string) bool {
```

Comment:
Could we put it to infoschema.go? I think it is like "IsMemoryDB", we can put these together.

Reply:
This function is removed, we forbid dropping tables in "mysql" database.